### DISCUSSION TITLE
PCAP Time filters not accurate?

### DISCUSSION THREAD

**Discussion by Sazerak** [Security-Onion-Solutions/securityonion#3191] [Unsupported Versions]
Sorry if this has already been answered before, did a search but didn't find anything.

When I grab PCAPs from SO and input a date/time filter, what I end up getting isn't quite accurate.

For example, when I put this in:
![pcap](https://user-images.githubusercontent.com/62576693/109505887-0cd05000-7a6b-11eb-8540-390f22f4b85c.png)

The PCAPs actually started at about 22:22. I also tried grabbing some PCAPs for a specific time around 4am a couple of days ago, and I could never get it to start at the right time, no matter what I tried. If I wanted 4:34am for example, I would put that in, but the first result would be from 4:35am. OK, so then I put 4:33am, and the logs started at 4:35am again. So I put in 4:32am, and all of a sudden the logs started at 4:32 and 5 seconds (or something like that, can't exactly remember).

Is this behaviour normal?

**Comment by weslambert** [comment]
It may  be that there was no applicable activity for that period of time 4:33-4:35.  You could try referencing the PCAP in /nsm/pcap on the relevant sensor, querying stenographer directly on that node for the given timeframe (ex. `docker exec -it so-sensoroni scripts/stenoquery.sh "$yourexpression"`).

https://github.com/google/stenographer#query-language
